receiver/azuremonitorreceiver/go.mod

```diff
@@ -0,0 +1,52 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/azuremonitorreceiver
+
+go 1.19
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.3.1
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.2.1
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/monitor/armmonitor v0.8.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v1.0.0
+	github.com/stretchr/testify v1.8.2
+	go.opentelemetry.io/collector v0.72.0
+	go.opentelemetry.io/collector/component v0.72.0
+	go.opentelemetry.io/collector/confmap v0.72.0
+	go.opentelemetry.io/collector/consumer v0.72.0
+	go.opentelemetry.io/collector/pdata v1.0.0-rc6
```

Comment:
These will need to be updated to match the latest version of the collector components